### DISCUSSION TITLE
Enable multiple options for minio login

### DISCUSSION THREAD

**Discussion by shivakumargowda12** [minio/minio#16824] [Q&A] [answered]
Hi, I have enabled OIDC login in minio.
During login page, its not showing options for login like User Credentials or STS. It shows only OIDC login option.

**Comment by vadmeste** [comment, marked as answer]
@shivakumargowda12 When OIDC is enabled, MinIO focuses on the external IDP and will not support internal users. This is currently expected.

**Comment by shivakumargowda12** [comment]
@vadmeste , Thanks for confirming. 
Will backend APIs work with normal users?
I feel,  still this should be addressed in future to support multiple login strategies.

**Reply by vadmeste** [reply]
> Will backend APIs work with normal users?

No, but you can service accounts associated to external accounts. Services accounts will not expire and they can be used by S3 apps.

**Reply by shivakumargowda12** [reply]
Thanks for the quick responses
